### DISCUSSION TITLE
[FeatureRequest] Add configuration option to show/hide sidebar when viewing a graph fullscreen.

### DISCUSSION THREAD

**Discussion by zegerhoogeboom** [grafana/grafana#38262] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

A configuration option to either show or hide the sidebar when viewing a graph fullscreen (using the "View" option).

**Why is this needed**:

In Grafana 7.4.x the sidebar was hidden when viewing a chart and in Grafana 7.5.x it is shown when viewing a chart as can be seen in these images:
![image](https://user-images.githubusercontent.com/5371096/115291238-caff8400-a154-11eb-81ea-82df2944de62.png)
![image](https://user-images.githubusercontent.com/5371096/115292201-f636a300-a155-11eb-9604-80d4dd3b0c3e.png)

The "kiosk" mode hides the sidebar but also hides the template variables, which I need.

I personally require the behavior from 7.4.x, hiding the sidebar. It matters for me because I'm looking at the same dashboard with a specific graph in "View" mode (giving me control of the template variables) in a grid of 5x6, controlled by external signage software (Xibo). This means in grafana 7.5.x the sidebar will show 5 times, taking up too much screen space.

I imagine other Grafana users would rather have the sidebar shown when viewing a graph full screen, therefore I believe it would be a good solution to make this behavior configurable.

**Comment by hugohaggmark** [comment]
@torkelo @dprokop is this a regression after React router work?

**Comment by dprokop** [comment]
@hugohaggmark can't be, react router is on master only currently

**Comment by hugohaggmark** [comment]
> @hugohaggmark can't be, react router is on master only currently

of course @dprokop 🙈

**Comment by torkelo** [comment]
No this is by design

**Comment by zegerhoogeboom** [comment]
Would it be acceptable to make this (globally) configurable @torkelo? Maybe an option can be added under the `panels` section like the following?

```
[panels]
;show_sidebar_in_fullscreen_panel = true
```

If this is not an interesting feature to put time into from the Grafana team but you would accept a PR I'm happy to give it a go.

**Comment by torkelo** [comment]
no option, maybe if there is huge demand. maybe a kiosk mode or option to hide/toggle sidebar

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
